Move maintenance flag lockfiles to a subdirectory so it can easily be shared
between nodes via a networked filesystem without sharing whole `var/` dir.

diff --git a/App/MaintenanceMode.php b/App/MaintenanceMode.php
index 5d3743a..1f0b385 100644
--- a/App/MaintenanceMode.php
+++ b/App/MaintenanceMode.php
@@ -22,12 +22,12 @@ class MaintenanceMode
      * DO NOT consolidate this file and the IP allow list into one.
      * It is going to work much faster in 99% of cases: the isOn() will return false whenever file doesn't exist.
      */
-    public const FLAG_FILENAME = '.maintenance.flag';
+    public const FLAG_FILENAME = 'global/.maintenance.flag';

     /**
      * IP-addresses file name
      */
-    public const IP_FILENAME = '.maintenance.ip';
+    public const IP_FILENAME = 'global/.maintenance.ip';

     /**
      * Maintenance flag dir
